scripts/src/owners/owners_file.py

```diff
@@ -66,9 +66,4 @@ def get_users_included(owner_data):
 
 
 def get_pgp_public_key(owner_data):
-    pgp_public_key = "null"
-    try:
-        pgp_public_key = owner_data["publicPgpKey"]
-    except Exception:
-        pass
-    return pgp_public_key
+    return owner_data.get("publicPgpKey", "")
```

Comment:
Have you checked the logic where this is called, and how it handles `""` vs. `"null"`? I know that we stuff "null" in some places, though I'm not certain as to why. To maintain compatibility here without diving deeper, I'd probably switch the default value if the key isn't found in that map to "null".


```suggestion
    return owner_data.get("publicPgpKey", "null")
```